style.css

```diff
@@ -0,0 +1,96 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  width: 375px;
+  height: 5410px;
+}
+
+header .container {
+  width: 100%;
+  margin-bottom: 0;
+}
+
+header .container .second-container {
+  display: flex;
+  justify-content: space-between;
+  padding: 20px 30px;
+}
+
+.icons div {
+  padding-right: 20px;
+}
+
+header .container .second-container div {
+  font-family: consolas, sans-serif;
```

Comment:
- [ ] Please use the appropriate font-family for all the elements. 👍 

## Your Design 
![crr15](https://user-images.githubusercontent.com/103386183/198584967-36cf10e2-c924-4dc5-aefa-1390a68c58a0.png)

## Figma Design
![crr16](https://user-images.githubusercontent.com/103386183/198585173-97f80f20-e110-46c6-9608-864678cd0c4a.png)